pages/blog/posts/understanding-lexical-dynamic-scopes.md

```diff
@@ -0,0 +1,248 @@
+---
+title: "Understanding JSON Schema Lexical and Dynamic Scopes"
+date: "2024-01-19"
+tags:
+  - Fundamentals
+  - Scopes
+type: Engineering
+cover: /img/posts/2024/understanding-lexical-dynamic-scopes/cover.webp
+authors:
+  - name: Juan Cruz Viotti
+    photo: /img/avatars/jviotti.webp
+    link: https://www.jviotti.com
+    byline: O'Reilly author, founder of Intelligence.AI and Sourcemeta
```

Comment:
I assume, like others, I may highlight `Intelligence.AI` and then open it in a new tab. The resulting site is not the one I'm looking for.

The non-www version of that URL is some weird gambling site... Assuming you have full control over the domain, might want to fix that and setup a redirect!

Reply:
Oh, what the hell. Yes, we own the domain. Not sure how this could be taking place there... I'll look into it cc @RonItelman

Reply:
Fixed